etl/steps/data/garden/un/2023-08-29/long_run_child_mortality.meta.yml

```diff
@@ -0,0 +1,96 @@
+dataset:
+  title: Long-run child mortality (Gapminder; UN IGME)
+  origins:
```

Comment:
Again, you shouldn't need to define origins here, they should already be present from the snapshot.